edgar_tool/constants.py

```diff
@@ -1390,10 +1389,9 @@
     },
 }
 
-# These groupings are not yet used, but are included for completeness
 TEXT_SEARCH_CATEGORY_FORM_GROUPINGS = {
     #    "Exclude insider equity awards, transactions, and ownership (Section 16 Reports)": ["-3","-4","-5"], # todo: work out how to exclude these
-    "All annual, quarterly, and current reports": [
+    "all_annual_quarterly_and_current_reports": [
```

Comment:
Why the change to snake case?

Reply:
I opted for snake case because it's easier for users to remember. Since it's case sensitive, even a minor error like a misplaced comma would cause an error.

A more sophisticated solution could be implemented using a basic NLP algorithm or regex, but I preferred to keep it simple. 

If you prefer I can revert it with the original keys.

Reply:
This is a good reason! You are right that we should definitely prioritize user experience

My only hesitation is if EDGAR updates the categories in the future. Updating the tool would take a bit more work. Currently the form groupings are just copied from some javascript objects. Perhaps the best solution is to revert for `TEXT_SEARCH_CATEGORY_FORM_GROUPINGS` but allow users to specify the snake_case options as well using the new mapping object you created `TEXT_SEARCH_FILING_VS_MAPPING_CATEGORIES_MAPPING`.

Then for the notebook application, we can use the more readable options for a drop-down. But it means command line users wont be disadvantaged. What do you think?

Reply:
Sounds good to me.

What if I revert `TEXT_SEARCH_CATEGORY_FORM_GROUPINGS` and use `TEXT_SEARCH_FILING_VS_MAPPING_CATEGORIES_MAPPING` to map the snake case input from the user into the "verbose" category?

Reply:
Perfect - thank you!